deeptrack/models/gans/pcgan.py

```diff
@@ -0,0 +1,152 @@
+import tensorflow as tf
+from tensorflow.python.keras.utils.generic_utils import default
+from .utils import as_KerasModel
+
+layers = tf.keras.layers
+
+
+@as_KerasModel
+class PCGAN(tf.keras.Model):
+    """Creates and compiles a conditional generative adversarial
+       neural network (cgan) with an additional perceptual discriminator. 
```

Comment:
Add a description of the different parameters

Reply:
Documentation added.

https://github.com/JesusPinedaC/DeepTrack-2.0/blob/2bb626ac48dd489d0ae9f8a12838df2d03801409/deeptrack/models/gans/pcgan.py#L9-L14

Reply:
Documentation added.

Reply:
:white_check_mark: Documentation added